tests/drivers/uart/uart_async_api/testcase.yaml

```diff
@@ -30,6 +30,18 @@ tests:
     harness_config:
       fixture: gpio_loopback
     depends_on: gpio
+  drivers.uart.async_api.wide:
+    filter: CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_ASYNC
+      and CONFIG_SERIAL_SUPPORT_WIDE_DATA and not CONFIG_UART_MCUX_LPUART
+    harness: ztest
+    harness_config:
+      fixture: gpio_loopback
+    depends_on: gpio
+    extra_configs:
+      - CONFIG_UART_WIDE_DATA=y
+      - CONFIG_UART_USE_RUNTIME_CONFIGURE=y
```

Comment:
What about using a "overlay-wide.conf" file with ` - OVERLAY_CONFIG="overlay-usb.conf"` here, see `drivers.uart.basic_api.cdc_acm` test variant. This would ease manual testing using west.

Reply:
Fixed